csrc/mmdeploy/backend_ops/snpe/server/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 gRPC authors.
```

Comment:
It seems to me that codes for `snpe_server` does not belong to `backend_ops` for `snpe`, and they should be put into a standalone directory and be triggered to  compile by a cmake flag.

Reply:
1. I have `mkdir service` and move `snpe/*`  to it
2. Compiling snpe service needs `grpc` which has nothing to do with `mmdeploy`.  Cross compile grpc is complicated, I do not want `mmdeploy cmake` to trigger it.